src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -25,6 +26,7 @@
 	protected ClinicianFacingPatientDashboardPage dashboardPage;
 	protected By patientHeaderId = By.cssSelector("div.identifiers span");
 	protected InitialSetupPage initialSetupPage;
+	protected PatientVisitsDashboardPage visitsDashboardPage;
```

Comment:
Added `PatientVisitsDashboardPage` to be a global variable for code reusability by child classes that extends Steps